geode-core/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/CreateRegionCommandPersistsConfigurationDUnitTest.java

```diff
@@ -310,10 +310,7 @@ public void createRegionAndValidateAllConfigIsPersistedForReplicatedRegion() {
         assertThat(attr.isStatisticsEnabled())
             .describedAs("Expecting statistics to be enabled for region " + name)
             .isTrue();
-        assertThat(attr.isCloningEnabled())
-            .describedAs("Expecting cloning to be enabled for region " + name
-                + " since compressor is provided")
-            .isTrue();
+        assertThat(attr.isCloningEnabled()).isNull();
```

Comment:
The semantics of this is kind of ugly (i.e. non-intuitive). We're testing that a method that returns a Boolean to return null. Yes,  I realize the declared return type (`Boolean`) could be null, but that confuses the intent of a method named `isXxxxx()`

The logic in `public boolean isConcurrencyChecksEnabled()` is more intuitive.

Reply:
these code are auto generated by jaxb. I believe `public boolean isConcurrencyChecksEnabled()` returns a `boolean` is because that attribute has a default defined in the xsd, which others don't.